packages/@n8n/nodes-langchain/nodes/mcp/McpClientTool/McpClientTool.node.ts

```diff
@@ -76,14 +76,31 @@ export class McpClientTool implements INodeType {
 		properties: [
 			getConnectionHintNoticeField([NodeConnectionTypes.AiAgent]),
 			{
-				displayName: 'SSE Endpoint',
-				name: 'sseEndpoint',
+				displayName: 'Endpoint',
+				name: 'endpointUrl',
 				type: 'string',
-				description: 'SSE Endpoint of your MCP server',
-				placeholder: 'e.g. https://my-mcp-server.ai/sse',
+				description: 'Endpoint of your MCP server',
+				placeholder: 'e.g. https://my-mcp-server.ai/mcp',
 				default: '',
 				required: true,
 			},
+			{
+				displayName: 'Server Transport',
+				name: 'serverTransport',
+				type: 'options',
+				options: [
+					{
+						name: 'Server Sent Events (Deprecated)',
+						value: 'sse',
+					},
+					{
+						name: 'HTTP Streamable',
```

Comment:
```suggestion
						name: 'Streamable HTTP',
```
As referred here: https://modelcontextprotocol.io/specification/2025-06-18/basic/transports#streamable-http